src/Tool-Base/MethodClassifier.class.st

```diff
@@ -49,53 +59,54 @@ MethodClassifier class >> buildPragmaMapping [
-		at: 'last' put: 'accessing';
-		at: 'matches' put: 'testing';
-		at: 'max' put: 'accessing';
-		at: 'maximum' put: 'accessing';
-		at: 'min' put: 'accessing';
-		at: 'minimum' put: 'accessing';
-		at: 'needs' put: 'testing';
-		at: 'new' put: 'instance creation';
-		at: 'parse' put: 'parsing';
-		at: 'print' put: 'printing';
-		at: 'remove' put: 'removing';
-		at: 'render' put: 'rendering';
-		at: 'requires' put: 'testing';
-		at: 'reset' put: 'initialization';
-		at: 'set' put: 'initialization';
-		at: 'should' put: 'asserting';
-		at: 'shouldnt' put: 'asserting';
-		at: 'signal' put: 'signalling';
-		at: 'sort' put: 'sorting';
-		at: 'test' put: 'tests';
-		at: 'total' put: 'accessing';
-		at: 'version' put: 'versions';
-		at: 'visit' put: 'visiting';
-		at: 'write' put: 'writing';
-		yourself
+		                 at: 'accept' put: 'visiting';
+		                 at: 'accepts' put: 'testing';
+		                 at: 'add' put: 'adding';
+		                 at: 'as' put: 'converting';
+		                 at: 'assert' put: 'asserting';
+		                 at: 'at' put: 'accessing';
+		                 at: 'baseline' put: 'baselines';
+		                 at: 'bench' put: 'benchmarking';
+		                 at: 'benchmark' put: 'benchmarking';
+		                 at: 'can' put: 'testing';
+		                 at: 'compile' put: 'compiling';
+		                 at: 'copy' put: 'copying';
+		                 at: 'deny' put: 'asserting';
+		                 at: 'first' put: 'accessing';
+		                 at: 'format' put: 'formatting';
+		                 at: 'from' put: 'instance creation';
+		                 at: 'has' put: 'testing';
+		                 at: 'includes' put: 'testing';
+		                 at: 'index' put: 'accessing';
+		                 at: 'initialize' put: 'initialization';
+		                 at: 'is' put: 'testing';
+		                 at: 'last' put: 'accessing';
+		                 at: 'matches' put: 'testing';
+		                 at: 'max' put: 'accessing';
+		                 at: 'maximum' put: 'accessing';
+		                 at: 'min' put: 'accessing';
+		                 at: 'minimum' put: 'accessing';
+		                 at: 'needs' put: 'testing';
+		                 at: 'new' put: 'instance creation';
+		                 at: 'parse' put: 'parsing';
+		                 at: 'print' put: 'printing';
+		                 at: 'remove' put: 'removing';
+		                 at: 'render' put: 'rendering';
+		                 at: 'requires' put: 'testing';
+		                 at: 'reset' put: 'initialization';
+		                 at: 'set' put: 'initialization';
+		                 at: 'should' put: 'asserting';
+		                 at: 'shouldnt' put: 'asserting';
+		                 at: 'signal' put: 'signalling';
+		                 at: 'sort' put: 'sorting';
+		                 at: 'test' put: 'tests';
+		                 at: 'total' put: 'accessing';
+		                 at: 'version' put: 'versions';
```

Comment:
@jecisc why not "versioning" instead of "versions" if we follow the Someth**ing** pattern

Reply:
This was already here, I just changed the formatting. It's possible there are more things to improve. I just wanted to align the release tests and the method classifier